packages/lending/src/adapter.ts

```diff
@@ -534,55 +556,55 @@ export class Adapter extends common.Web3Toolkit {
             // 1-5-1. the supply amount is the swap quotation output
             supplyInput = swapQuotation.output;
```

Comment:
Wondering if we need to add a try-catch for Ln.555

Reply:
Fixed~